### PR TITLE
feat: handle response properly on askai when length is over 2k chars

### PR DESCRIPTION
**Pull request by 7ez**
the shit code pr is here!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -360,7 +360,20 @@ async def askai(self, ctx: Context) -> None:
...
+                with open(f"/tmp/{response.openai_id}.txt", "w") as f:
+                    f.write(response_text)
+            except Exception:
+                await ctx.send("Failed to write the response to file.")
+                return
+
+            response_file = discord.File(f"/tmp/{response.openai_id}", "response.txt")
+            await ctx.send(f"Spent {cents_spent:.5f}¢ ({response.usage.total_tokens} tokens) to produce result:", file=response_file)
+            with contextlib.suppress(BaseException):
+                os.remove(f"/tmp/{response.openai_id}.txt")
```
Comment (cmyui) [inline main.py:374]: hey hey thanks for the pr to fix my todo :D 

i think we may be able to do a bit better - do you know if it's possible to use an in-memory file-like-object like io.bytesIO with discord.File? that would be the ideal solution, as we wouldn't have to write to disk

if that's not possible, could you use a [tempfile](https://docs.python.org/3/library/tempfile.html) with a context manager? that's a bit better than manually writing to /tmp

Comment (7ez) [reply main.py]: fixed

**Comment by minisbett** [comment]
why don't you just send 2k characters at once, splitting up the >2k character reply into multiple discord messages...

**Comment by 7ez** [comment]
because its better to just send it as file

**Comment by 7ez** [comment]
its not like youre going to generate 8mb of text

File: requirements.txt [hunk 2]
```diff
@@ -13,3 +13,4 @@ py-cpuinfo==8.0.0
 timeago==1.0.15
 typing_extensions==4.2.0
 yarl==1.7.2
+openai==0.25.0
```
Comment (cmyui) [inline requirements.txt:16]: nit: out of order, but made me realize pre-commit CI isn't setup

**Review by cmyui — APPROVED** [review]

**Comment by cmyui** [comment]
ty for good pr

**Pull request merged**